packages/components/src/components/vertical-tab/vertical-tab-content-container.jsx

```diff
@@ -1,9 +1,9 @@
 import classNames        from 'classnames';
 import React             from 'react';
 import { Route, Switch } from 'react-router-dom';
-import { Icon }          from '@deriv/components';
+import Icon              from 'Components/icon';
 
-class VerticalTabContentContainer extends React.PureComponent {
+export default class VerticalTabContentContainer extends React.PureComponent {
     render() {
         const {
             action_bar,
```

Comment:
Please prefix all classnames in this file with `dc-` (with updates to their respective stylesheet).